curvefs/src/mds/mds_service.cpp

```diff
@@ -215,6 +215,7 @@ void MdsServiceImpl::AllocateS3Chunk(
     ::curvefs::mds::AllocateS3ChunkResponse* response,
     ::google::protobuf::Closure* done) {
     brpc::ClosureGuard guard(done);
+    LOG(INFO) << "start to allocate chunkId";
```

Comment:
If this log necessary?

Reply:
> 

Sometimes need to know the time to allocate chunkid